pkg/convert/jfr/parser.go

```diff
@@ -203,3 +205,24 @@ func frames(st *parser.StackTrace) []string {
 	}
 	return frames
 }
+
+// jdk/internal/reflect/GeneratedMethodAccessor31
+var generatedMethodAccessor = regexp.MustCompile("^(jdk/internal/reflect/GeneratedMethodAccessor)(\\d+)$")
+
+// org/example/rideshare/OrderService$$Lambda$669.0x0000000800fd7318.run
+var lambdaGeneratedEnclosingClass = regexp.MustCompile("^(.+\\$\\$Lambda\\$)\\d+[./](0x[\\da-f]+|\\d+)$")
+
+func mergeJVMGeneratedClasses(frame string) string {
+	frame = generatedMethodAccessor.ReplaceAllString(frame, "${1}_")
+	frame = lambdaGeneratedEnclosingClass.ReplaceAllString(frame, "${1}_")
```

Comment:
I appended `_` to the merged frame name, I want to put there something more meaningful, like `_merged` or `_$$merged` or `_$$redacted` `_$$pyroscope_merged` 
What do you think?

Reply:
Same here